scripts/deploy_staking_escrow.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+from brownie import StakingEscrow, config
+from scripts.utils import CURRENT_NETWORK, LOCAL_BLOCKCHAIN_ENVIRONMENTS, get_account
+
+
+def deploy_mocks():
+    """This function should deploy nucypher_token and t_staking and return the
+    corresponding contract addresses"""
+    nucypher_token = None
+    t_staking = None
+    return nucypher_token, t_staking
+
+
+def main(id=None):
+    deployer = get_account(id)
+    network_config = config["networks"][CURRENT_NETWORK]
+    if CURRENT_NETWORK in LOCAL_BLOCKCHAIN_ENVIRONMENTS:
+        nucypher_token, t_staking = deploy_mocks()
+        work_lock = "0x0000000000000000000000000000000000000000"
```

Comment:
How can this work later since `nucypher_token` and `t_staking` are both `None`?

Reply:
#47 is an issue for completing the `deploy_mocks` function

Reply:
Finished